### DISCUSSION TITLE
Nach Neuinstallation kein Access Point

### DISCUSSION THREAD

**Discussion by MrBS90602** [tbnobody/OpenDTU#2220] [Q&A]
Hallo zusammen,
auf meinem ESP32 lief eine alte OpenDTU Version von 22. Diese konnte ich nicht aktualisieren. Deshabl habe ich sie mit der aktuellen Version geflashed (ESP32 Flasch Download Tool). Erase war erfolgreich, und Beschreiben ebenfalls erfolgreich. Danach ein reboot, aber ich finde keinen WLAN AccessPoint.. Am ESP32 flacker eine rote LED sehr schnell (5. von links neben Boot-Button).
Was kann ich tun?
Danke

**Comment by ghost** [comment]


**Reply by ghost** [reply]
ich flashe meist mit dem [ESP Tool](https://espressif.github.io/esptool-js/).
Oder nimm den [OpenDTU Web Flasher](https://www.opendtu.solar/firmware/webinstall/).

**Reply by MrBS90602** [reply]
@schneeer 
Besten Dank, mit dem OpenDTU Web Flasher hat es letztlich geklappt. Leider habe ich die Konfiguration nicht gesichert, muss somit alles neu machen. Aber zumindest habe ich Dank Deiner Hilfe eine aktuelle Version der OpenDTU.
Nochmal vielen Dank für Deine Unterstützung
VG
Bernhard

**Reply by mroenne2022** [reply]
Mit der Vorgehensweise hast du das Partitionsproblem gleich mit behoben.

**Reply by ghost** [reply]
Hi, 

gerne.

Das war ja auch der Sinn.
Wie nachfolgend geschrieben wurde, steht das auch groß und breit auf der Startseite da.
Im ersten Moment kommt da keiner drauf, und liest auch keiner.
Wichtig ist, das die vorgeschlagene Vorgehensweise hilfreich ist.
Das unterm Strich die Konfiguration flöten ging, war letztendlich zu erwarten, der ESP hatte sich ja durch den Updateversuch schon "halb verabschiedet".
Hätte vorher gesichert werden können. Wer denkt bei einem Update an größere Probleme?

**Reply by MrBS90602** [reply]
Versuuch macht kluuch :-)
Da stecke ich halt nicht so tief in der Materie drin...

Aber jetzt habe ich das Problem, dass ich zwar meine Hoymiles eingetragen habe, die OpenDTU aber von keinem Daten empfängt!

**Reply by mroenne2022** [reply]
Hast du die Punkte in den WR Einstellungen aktiviert? Foto !
![Screenshot_20240822_142315_Chrome](https://github.com/user-attachments/assets/e3b25020-6e14-417d-81df-99398a8ae407)

**Reply by mroenne2022** [reply]
Und hat er eine IP im WLAN bekommen und somit die Zeit synchronisiert?

**Reply by mroenne2022** [reply]
Und ist das Funkmodul aktiviert (NRF.....) ?  
![Screenshot_20240822_142815_Chrome](https://github.com/user-attachments/assets/c186c2b6-7208-4a6b-9172-bb530096dba6)

**Reply by MrBS90602** [reply]
Hallo,
Sieht bei mir alles etwas anders aus. Aber in der Tat steht bei Funkmodul Chip Status nicht verbunden. Das Welle eine Erklärung. Aber warum? Vor dem Upgrade war ja alles ok?
VG

**Reply by MrBS90602** [reply]
Hier noch ein paar Screenshots 
![image](https://github.com/user-attachments/assets/9f6b98df-5168-4d10-8ba4-6ad5cc7ac201)
![image](https://github.com/user-attachments/assets/ea0388a0-3a82-4e3a-aa14-33e1b716c9c6)
![image](https://github.com/user-attachments/assets/23a02ac4-8617-4631-a825-72c528efc717)
![image](https://github.com/user-attachments/assets/480219ff-d5e7-4bc3-a489-64741317bf21)

**Reply by mroenne2022** [reply]
Aber die aktuelle Version ist es nicht (24.4.24 !?) 
Allerdings sollte sie dennoch funktionieren. Hast du die pin_mapping geladen ? Es sollte die nodemcu_esp32 sein.
Die bekommst du hier https://github.com/tbnobody/OpenDTU/tree/master/docs/DeviceProfiles

**Reply by MrBS90602** [reply]
Aber an der Verkabelung habe ich nichts verändert. Vor dem Update war es ja in Ordnung. Aber ich prüfe es
Danke

**Reply by mroenne2022** [reply]
1) Du hast ja eine generic_esp32 geladen, dazu benötigt man ein Devicefile 
2) Es ist ja nicht die aktuelle Version vom 5.8. installiert ,sondern eine aus April, das beides solltest du mal untersuchen.

**Comment by mroenne2022** [comment]
Ich glaube eher das du dieses Problem hast mit dem zu kleinen Filesystem ...
Ab März 2023 mussten die Partitionen verändert werden. 
Evtl mal diese Doku befolgen : https://github.com/tbnobody/OpenDTU/blob/master/docs/UpgradePartition.md

![Screenshot_20240822_132735_Chrome](https://github.com/user-attachments/assets/205cd1d2-31ad-43c2-b13f-80e012269f1e)

**Comment by ghost** [comment]


**Reply by mroenne2022** [reply]
@schneeer : 
Sehr gut und ausführlich beschrieben, damit sollte er es hinbekommen hoffe ich. So gut hätte ich es auf die "Schnelle" nicht beschreiben können ;-)

**Comment by MrBS90602** [comment]
@schneeer 
Vielen Dank, auch an @mroenne2022 , für diese ausführliche Anleitung. Ich denke, ich sollte es damit hinbekommen. Über das Ergebnis werde ich in jedem Fall berichten, wird aber einige Zeit dauern, da ich ab morgen "offline" bin.
VG
